tests/apichecks/testdata/exceptions/missingrefs.txt

```diff
@@ -10,7 +10,12 @@
 [refs] crd=apigeenataddresses.apigee.cnrm.cloud.google.com version=v1alpha1: field ".spec.instanceId" should be a reference
 [refs] crd=appengineflexibleappversions.appengine.cnrm.cloud.google.com version=v1alpha1: field ".spec.serviceAccount" should be a reference
 [refs] crd=appenginestandardappversions.appengine.cnrm.cloud.google.com version=v1alpha1: field ".spec.serviceAccount" should be a reference
+[refs] crd=batchjobs.batch.cnrm.cloud.google.com version=v1alpha1: field ".spec.allocationPolicy.instances[].policy.bootDisk.image" should be a reference
+[refs] crd=batchjobs.batch.cnrm.cloud.google.com version=v1alpha1: field ".spec.allocationPolicy.instances[].policy.disks[].newDisk.image" should be a reference
 [refs] crd=batchjobs.batch.cnrm.cloud.google.com version=v1alpha1: field ".spec.allocationPolicy.serviceAccount" should be a reference
+[refs] crd=batchjobs.batch.cnrm.cloud.google.com version=v1beta1: field ".spec.allocationPolicy.instances[].policy.bootDisk.image" should be a reference
+[refs] crd=batchjobs.batch.cnrm.cloud.google.com version=v1beta1: field ".spec.allocationPolicy.instances[].policy.disks[].newDisk.image" should be a reference
+[refs] crd=batchjobs.batch.cnrm.cloud.google.com version=v1beta1: field ".spec.allocationPolicy.serviceAccount" should be a reference
```

Comment:
Yeah, I think this is correct ... both image and serviceAccount should be references?